### PR TITLE
Handle import for users with no invitation email

### PR DESCRIPTION
**Pull request by michieldewilde**
## Description of the change

- Make invitation email optional
- Handle invitation email being not provided during create

## Type of change

- [x] Bug fix (non-breaking change that fixes an issue)

### Development

- [x] Lint rules pass locally
- [x] The code changed/added as part of this pull request has been covered with tests
- [x] All tests related to the changed code pass in development
- [x] Examples for new resources and data sources have been added
- [x] Default values have been documented in the description (e.g., "Dummy: (Boolean) Blah blah. Defaults to `false`.)
- [ ] If the action fails that checks the documentation: Run `go generate` to make sure the docs are up to date

### Code review

- [x] This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [x] Pull Request is no longer marked as "draft"
- [x] Reviewers have been assigned
- [ ] Changes have been reviewed by at least one other engineer

### CODE REVIEW

File: docs/resources/user.md [hunk 1]
```diff
@@ -17,10 +17,13 @@ description: |-
 
 ### Required
 
-- `invitation_email` (String) Email of the user. Used for sending an invitation.
 - `policy` (Block List, Min: 1) (see [below for nested schema](#nestedblock--policy))
 - `username` (String) Username of the user
 
+### Optional
+
+- `invitation_email` (String) `invitation_email` will be used to send an invitation to specified email addressThis property is required, when creating a new user.This property is optional, when importing an existing user.
```
Comment (KNiepok) [inline docs/resources/user.md:25]: Some grammarly fixes: 

- `invitation_email` (String) `invitation_email` will be used to send an invitation to a specified email address. This property is required when creating a new user. This property is optional when importing an existing user.

**Review by marcinwyszynski — APPROVED** [review]

File: docs/resources/user.md (hunk 1 above)
Comment (marcinwyszynski) [inline docs/resources/user.md:25]: Weird formatting, missing whitespaces. I think you will want to fix it in the original description, then regenerate the doc?

Comment (michieldewilde) [reply docs/resources/user.md:25]: looks like it

**Review by KNiepok — APPROVED** [review]
minor comment

**Review by mwasilew2 — APPROVED** [review]

**Pull request merged**